### PR TITLE
Migrate BigQuery and classpath driver tests to GitHub actions

### PR DESCRIPTION
**Pull request by nemanjaglumac**
Note:
Code can probably be a bit dry-er, but let's worry about that later when we move all tests and when we have a good overview of all tests.

### CODE REVIEW

**Review by flamber — APPROVED** [review]

File: .github/actions/test-driver/action.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ inputs:
   junit-name:
     required: true
     default: 'driver'
+  test-args:
+    required: false
```
Comment (flamber) [inline .github/actions/test-driver/action.yml:7]: Why this change?

Comment (nemanjaglumac) [reply .github/actions/test-driver/action.yml:7]: Because we need to provide the additional arguments to how we invoke the command and it wasn't possible to do that before.

All other steps only assume this will be run always `clojure -X:dev:ci:ee:ee-dev:drivers:drivers-dev:test`
